src/index.ts

```diff
@@ -140,7 +142,8 @@ export interface CreateV2Options {
 }
 
 const defaultCreateOptions: CreateOptions = {
-  v1Compatible: true
+  v1Compatible: true,
+  lifetimeNs: DEFAULT_TTL
```

Comment:
For consistency with Kubo/Boxo, it's probably sensible to set the default lifetime to 48 hours, which is also the DHT record expiration window

https://github.com/ipfs/boxo/blob/2559ec2c7b93ac6905fe83d7a4b38b8e9a703b0a/ipns/defaults.go#L10-L16

Reply:
lifetimeNs is a poor name that was previously used and I inherited.  I would be happy to modify the const and change the name to something like ttlNs which is what the value is actually for.

If desired we could keep the value as a default and simply set the TTL to be 1hr which is the default suggested per spec and the default in Kubo.

Reply:
I am going to advocate for allowing the TTL to be set externally as most DNS applications allow you to set the TTL externally and the Kubo CLI also does.

Of note the Kubo CLI allows setting the TTL via -t or --lifetime.  In the case of the js-ipns library the current lifetime passed is used to set the DHT expiration instead of affect the TTL.

Reply:
> I am going to advocate for allowing the TTL to be set externally as most DNS applications allow you to set the TTL externally and the Kubo CLI also does.

Me too. We're on the same page. My comment may have been confusing. I thought it may make sense to add this as another argument, but we already have enough arguments. I'm ok with this going into the CreateOptions.

My suggestion would be to name it ttl and suffix the unit for nicer ergonomics (as you've done already adding the `Ns` suffix).